### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by sam2if**
In this project:

- I have created a new game with the name of my choice by using the API.
- I have Implemented the "Refresh" button that will get all scores for a game created by me from the API (receiving data from the API and parsing the JSON).
- I have Implemented the form "Submit" button that will save a score for a game created by me (sending data to the API

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
 
-const add = document.querySelector('.add');
-add.addEventListener('click', added);
\ No newline at end of file
+const refresh = document.getElementById('refresh');
+const form = document.getElementById('form');
+const gameApi = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/GDUN7EKtNyCRpmru02cT/scores';
+
+refresh.addEventListener('click', () => {
+  getScores(gameApi).then((data) => showScores(data.result));
```
Comment (AnselemOdims) [inline src/index.js]: - [ ] Good job working on this but please kindly ensure to use `async` and `await` for asynchronous function calls.

File: src/module/add.js [hunk 2]
```diff
@@ -11,22 +11,28 @@ class Scores {
   }
 }
 
-function display(player) {
+const order = () => {
+  fetch('https://jsonplaceholder.typicode.com/todos/1')
+    .then((response) => response.json())
+    .then((json) => console.log(json.title));
```
Comment (AnselemOdims) [inline src/module/add.js]: - [ ] As pointed out above, please kindly use `async` and `await`

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @sam2if , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rotimiazeez — APPROVED** [review]
Hi @sam2if 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job implementing the required changes 👍🏽
- Well organized codebase 👏🏽
- App function as it should 😉
- No linter error ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**